packages/vhd-lib/merge.integ.spec.js

```diff
@@ -28,11 +29,13 @@ afterEach(async () => {
   await pFromCallback(cb => rimraf(tmpDir, cb))
 })
 
-async function createRandomFile(name, sizeMb) {
-  await execa('bash', [
-    '-c',
-    `< /dev/urandom tr -dc "\\t\\n [:alnum:]" | head -c ${sizeMb}M >${name}`,
-  ])
+async function createRandomFile(name, size) {
+  await fromEvent(
+    createReadStream('/dev/urandom', { end: size * 1024 * 1024 - 1 }).pipe(
```

Comment:
It's still a dependency on the environment, and I think pretty easy to do without:

```js
const createRandomStream = asyncIteratorToStream(function *(size) {
  while (size-- > 0) {
    yield Math.floor(Math.random() * 256)
  }
})

const input = createRandomStream(size * 1024 * 1024 - 1)
```